letsencrypt/translations/en.yaml

```diff
@@ -37,5 +37,11 @@ configuration:
   dns:
     name: DNS
     description: DNS Provider configuration
+  keytype:
```

Comment:
`keytype` already exist a bit further up. You can move it down to keep it nearby `elliptic_curve`, but we shouldn't have it twice.

Reply:
Missed, that, fixed now.